Completamento UC-1 e UC-2

Carraro-Riccardo opened the pull request:
Link alla issue di Jira:
https://error418swe.atlassian.net/browse/DOC-182?atlOrigin=eyJpIjoiNGQyZmMyNWQ4YjBmNGRkMmI4N2RjZmMzZDRmNzkyOTgiLCJwIjoiaiJ9

https://error418swe.atlassian.net/browse/DOC-183?atlOrigin=eyJpIjoiM2EyMjI5OTJjZjBjNGE3ZThjMzViMjQ4NjJhY2IxNWIiLCJwIjoiaiJ9
Note:
Erroneamente ho completato anche l'uc2
### ✏️ Checklist

- [x] documento testato;
- [x] titolo della PR conforme;
- [x] designato almeno un verificatore.

Carraro-Riccardo committed:
Completamento UC-1 e UC-2

Erroneamente avevo già completato anche la UC-2

FakeDaemon commented on 2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ:
```diff
@@ -106,12 +111,11 @@ $bold("Scenario: ")$
-
-==== UC-1.1.1 Visualizzazione errore lettura del file SVG dovuto a file privo di informazioni
+\
+==== UC-1.1.1.1 Visualizzazione errore file privo di informazioni
```
compilando mi mostra due volte UC 1.1.1.1

Carraro-Riccardo replied:
![image](https://github.com/Error-418-SWE/Documenti/assets/77143275/be626394-155a-494e-9a0d-ee5d7cf245b2)


Dove esattamente?

FakeDaemon replied:
![image_2023-12-03_104622088](https://github.com/Error-418-SWE/Documenti/assets/91839513/279a0c92-5aaa-4f17-99e9-bfbf66866d7e)

FakeDaemon replied:
nel titolo hai messo === UC-1.1.1.2, devi togliere il numero della UC che viene calcolato dalla function

FakeDaemon commented on 2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ:
```diff
@@ -129,10 +133,8 @@ $bold("Postcondizioni: ")$
-
-==== UC-1.1.2 Visualizzazione errore lettura del file SVG dovuto a informazioni fornite incongruenti
+\
+==== UC-1.1.1.2 Visualizzazione errore informazioni del file incongruenti
```
compilando mostra due volte UC-1.1.1.2

Carraro-Riccardo replied:
Come nella risposta precedente

FakeDaemon replied:
come sopra

FakeDaemon commented on 2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ:
```diff
@@ -192,9 +194,10 @@ $bold("Postcondizioni: ")$
-
-
-== UC-3 Modifica ambiente 3d
+\
```
più che uno spazio inserirei proprio un newpage in modo da visualizzare titolo e immagine insieme

Carraro-Riccardo replied:
Si lo avevo notato, ma magari aggiungevamo qualcosa nell'introduzione e quindi non serviva mettere nessun new page... in ogni caso al momento lo metto e in caso in futuro si toglie

FakeDaemon commented on 2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ:
```diff
@@ -277,30 +278,49 @@ $bold("Scenario: ")$
 $bold("Precondizioni: ")$
--l'attività di modifica di uno scaffale deve essere stata attivata;
--l'inserimento dei valori per la modifica deve essere avvenuto.
+- l'attività di spostamento di uno scaffale deve essere stata attivata;
```
non mi suona benissimo come frase, forse è meglio un " deve essere in corso lo spostamento dello scaffale" o "avviata l'attività di spostamento dello scaffale"

actions-user committed:
Aggiunti dettagli di rilascio

actions-user committed:
Aggiunti dettagli di rilascio

actions-user committed:
Aggiunti dettagli di rilascio

actions-user committed:
Aggiunti dettagli di rilascio

Carraro-Riccardo committed:
correzione conversation

- aggiunto pagebrake per lo use case 2
- riformulazione precondizioni "UC visualizzazione errore modifica dell'ambiente"

Carraro-Riccardo committed:
Merge branch 'DOC-182-Completare-UC-1' of https://github.com/Error-418-SWE/Documenti into DOC-182-Completare-UC-1

actions-user committed:
Aggiunti dettagli di rilascio

actions-user committed:
Aggiunti dettagli di rilascio

actions-user committed:
Aggiunti dettagli di rilascio

Carraro-Riccardo committed:
Rimozione duplicato nome UC

rimozione duplicato nome UC nei paragrafi 1.1.1.1 e 1.1.1.2
